### ISSUE TITLE
Cannot open G09 log file in version 1.6.6

### ISSUE THREAD

**Issue by jalemkul** [portobiocomp/molUP#5]
Hi,
Thanks for your help with the previous sed issues, but unfortunately I'm having similar problems with the latest version of molUP. While I can open a G09 output file (geometry optimization and frequency calculation), any attempt to animate the frequencies leads to an error (below). This problem is the same on Ubuntu 16.04 and MacOS Catalina, so I do not think it is platform-specific (my Mac has gsed and ggrep, which the documentation says should work and solved my previous issue).

A simple test case (water) is attached. The file loads but frequencies will not animate. The error message is quoted below

```
invalid bareword "sed"
in expression "sed: -1";
should be "$sed" or "{sed}" or "sed(...)" or ...
invalid bareword "sed"
in expression "sed: -1";
should be "$sed" or "{sed}" or "sed(...)" or ...
    (parsing expression "sed: -1")
    invoked from within
"expr [lindex $freq 0] -1 "
    (procedure "molUP::animateFreq" line 19)
    invoked from within
"molUP::animateFreq $molUP::freqVectorsList $molUP::animationFreq $molUP::displacement "none""
    (procedure "molUP::selectFreq" line 22)
    invoked from within
"molUP::selectFreq"
    (command bound to event)
```
[water_opt_freq_mp2.log](https://github.com/BioSIM-Research-Group/molUP/files/6109462/water_opt_freq_mp2.log)

**Comment by henriquefer** [comment]
Hello,
You are welcome. I am sorry for the issue you are experiencing. I have check the code, and it was an issue that prevents the animation of frequencies in some cases. 
I have fix that issue and released a new version (v1.6.7). Please, update your molUP version and let me know if it fixes the problem.

Thank you

**Comment by jalemkul** [comment]
Problem solved, thank you for the quick fix!
